Wikipedia/Code/WMFArticleFetcher.h

```diff
@@ -22,6 +23,8 @@ extern NSString* const WMFArticleFetcherErrorCachedFallbackArticleKey;
 - (void)cancelFetchForPageTitle:(MWKTitle*)pageTitle;
 - (void)cancelAllFetches;
 
+@property (nonatomic, strong) AFHTTPSessionManager* operationManager;
```

Comment:
I see that you did this for a test which is good, but we shouldn't leave this as a read/write property in the public API.

What we have been typically doing is:
1. Define this property in a class extension in the implementation file (`@property (nonatomic, strong) AFHTTPSessionManager* operationManager;`)
2. Create a separate category file for testing… like  WMFArticleFetcher_Tests.h
3. In the header of that category declare this method again (`@property (nonatomic, strong) AFHTTPSessionManager* operationManager;`)
4. Import this category header into your test.

Make sense?